proto/scene.proto

```diff
@@ -62,8 +62,16 @@ message RenderOptions {
   int32 height = 3;
 }
 
-// Representation of cube 
-message Cube {}
+// A Cube contains two fields:
+// topLeftForward: Vector3, represents the top left forward point of the cube in space. 
+// bottomRightBackward: Vector3, represents the bottom right backward point of the cube in space. 
+// Using only this 2 point you can define any cube in space. You need to wrap this type into SceneObject if you need to apply transform, rotate or scale. 
+message Cube {
+// topLeftForward point
+ Vector3 tlf = 1;
+ // bottomRightBackward point
+ Vector3 brb = 2;
+}
```

Comment:
Cube is defined by transform. See #16 for an updated comment.
I think we can remove this comment and use the one from #16

Reply:
Yes, totally agree, that description is better. This one is outdated.